frontend/src/components/Pages/ErrorFallback.tsx

```diff
@@ -0,0 +1,16 @@
+import { UnauthorizedPage } from './UnauthorizedPage/UnauthorizedPage'
+
+export const ErrorFallback = (error: Error) => {
+    switch (error.message) {
+        case '403 - ': {
+            return <UnauthorizedPage />
+        }
+        default: {
+            return (
+                <>
+                    Type = {error.name} - {error.message} - {error.cause}
```

Comment:
This default message has too much information, suggesting reducing it
```suggestion
                    An unknown error has occured
```

Reply:
Is there something between exposing too much information and "An unknown error has occurred"?

Reply:
I think exposing no information we can't control is the best for now, and we have to add more specific information pages for other error cases